modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase

```diff
@@ -147,12 +147,41 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-6027. EditAnImage"
-	@ignore = "true"
 	@priority = "3"
 	test EditAnImage {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6027 EditAnImage
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
+
+		CPCommerceCatalog.newProductsEntry(
+			productCatalog = "Master",
+			productName = "Simple T-Shirt",
+			productType = "Simple");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Media");
+
+		CommerceProductImages.addProductImages(
+			imageTitles = "Black",
+			validationImage = "true");
+
+		Click(
+			key_value = "Commerce_Black.jpg",
+			locator1 = "CommerceEntry#BODY_VERTICAL_ELLIPSIS");
+
+		MenuItem.click(menuItem = "Edit");
+
+		SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+		Type(
+			locator1 = "TextInput#TITLE",
+			value1 = "Commerce_Black_Edit");
+
+		Button.clickSaveAsDraft();
```

Comment:
I'm sorry I wrote wrong in the ticket when I said save I meant to click on the publish button